packages/cipherstash-proxy/src/config/tandem.rs

```diff
@@ -254,18 +259,31 @@ impl Default for PrometheusConfig {
     }
 }
 
+// static RE: LazyLock<Regex> =
+//     LazyLock::new(|| Regex::new(r"missing field `([^`]+)`(.+)`([^`]+)`").unwrap());
```

Comment:
Is this just an older version hanging around?

Reply:
I'm also wondering if it's better to check for "missing" (or "missing field") if the code assumes it's a missing field. This may be less confusing if anybody adds "invalid field" or something in the future, but might not be helpful either, so I'm not sure.

Reply:
I actually pulled the config crate to see if I could add more structure to the errors, but it seemed like it would require some pretty invasive changes.
Added extra tests instead. If the underlying errors change, we should catch.